win32ss/user/ntuser/window.c

```diff
@@ -1040,8 +1041,15 @@ VOID FASTCALL IntLinkHwnd(PWND Wnd, HWND hWndPrev)
         }
 
         if (Wnd == WndInsertAfter)
-            ERR("IntLinkHwnd -- Trying to link window 0x%p to itself!!\n", Wnd);
-        IntLinkWindow(Wnd, WndInsertAfter);
+        {
+            ERR("Trying to link window 0x%p to itself\n", Wnd);
+            ASSERT(WndInsertAfter != Wnd);
+            FIXME("IntUnlinkWindow() was already called. Continuing as is seems wrong\n");
```

Comment:
Placing the FIXME after the assert doesn't seem to make a lot of sense to me. Why not adding that text to the ERR() above?

Reply:
I am not sure about this fixme, but the rest lgtm

Reply:
I demoted the FIXME line to a comment. That should be enough for this PR.